projects/packages/my-jetpack/_inc/components/stats-section/count-comparison-card.jsx

```diff
@@ -35,54 +35,66 @@ export const percentCalculator = ( part, whole ) => {
  * @param {number}          props.previousCount - Previous count.
  * @param {React.ReactNode} props.icon          - Icon to display.
  * @param {React.ReactNode} props.heading       - Card heading.
+ * @param {string}          props.as            - Card root element type.
+ * @param {string}          props.srText        - Text for screen readers.
  * @return {object} CountComparisonCard React component.
  */
-const CountComparisonCard = ( { count = 0, previousCount = 0, icon, heading } ) => {
+const CountComparisonCard = ( {
+	count = 0,
+	previousCount = 0,
+	as = 'div',
+	icon,
+	heading,
+	srText,
+} ) => {
 	const difference = subtract( count, previousCount );
 	const differenceMagnitude = Math.abs( difference );
 	const percentage = Number.isFinite( difference )
 		? percentCalculator( differenceMagnitude, previousCount )
 		: null;
 
 	return (
-		<Card className={ styles[ 'stats-card' ] }>
-			{ icon && <div className={ styles[ 'stats-card-icon' ] }>{ icon }</div> }
-			{ heading && <div className={ styles[ 'stats-card-heading' ] }>{ heading }</div> }
-			<div className={ styles[ 'stats-card-count' ] }>
-				<span
-					className={ styles[ 'stats-card-count-value' ] }
-					title={ Number.isFinite( count ) ? String( count ) : undefined }
-				>
-					{ formatNumber( count ) }
-				</span>
-				{ difference !== null ? (
+		<Card className={ styles[ 'stats-card' ] } as={ as }>
+			<span className="screen-reader-text">{ srText }</span>
+			<div aria-hidden="true">
```

Comment:
Hide existing content from screen readers and add text that's visible to screen readers only.